clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/model/ServerGroup.groovy

```diff
@@ -122,6 +123,7 @@ interface ServerGroup {
   @JsonIgnore
   ImageSummary getImageSummary()
 
+  @JsonPropertyOrder(alphabetic = true)
```

Comment:
I had to look for it too - https://docs.spring.io/spring-boot/docs/current/reference/html/howto-spring-mvc.html#howto-customize-the-jackson-objectmapper